### DISCUSSION TITLE
mod_custom Module class not added in html code

### DISCUSSION THREAD

**Discussion by joris85** [joomla/joomla-cms#34229] [Q&A] [answered]
### Steps to reproduce the issue
Create a new mod_custom module
In advanced tab add module class
Publish it on a location that is active
Check html in frontend to see if this is added

Tried Cassiopeia theme and my own theme

### Expected result
I expect that there is a class added in the mod_custom div or on other location

### Actual result
I do not see the class coming back in the frontend.

### System information (as much as possible)
Joomla 4 beta 7
Php 7.4

**Comment by ReLater** [comment, marked as answer]
Please check:

It depends on the module position markup in your index.php if the class gets inserted.

If you use a position with `style="none"` like
`<jdoc:include type="modules" name="search" style="none" />`

you have to select e.g. 'html5' as Module Style in your module settings.
Or you can use a custom module override (Layout) where the class is used then.

**Comment by ReLater** [comment]
Thats the related pr : https://github.com/joomla/joomla-cms/pull/17447

**Comment by brianteeman** [comment]
The explanation by @ReLater is correct.

For the record I think it is wrong that the template has any position with style = none and that the minimum module chrome should support the standard params such as title, class

**Comment by joris85** [comment]
Thanks for the support guys! 

I agree to Brian settings should always be working, also with none (: 

I do understand it is now only one time. I had my own mod chrome for that in J3 to use bootstrap 3 classes.

I thought i copied the html5 and created a copy as default in my own template with no changes.
I now see i used another file 😇

I now see that that works, my bad here 😋
